bitnami/grafana-mimir/values.yaml

```diff
@@ -0,0 +1,4188 @@
+      # queriers during non-sharded queries.
+      max_concurrent: 16
+    query_scheduler:
+      # Increase from default of 100 to account for queries created by query sharding
+      max_outstanding_requests_per_tenant: 800
+    server:
+      grpc_server_max_concurrent_streams: 1000
+      grpc_server_max_connection_age: 2m
+      grpc_server_max_connection_age_grace: 5m
+      grpc_server_max_connection_idle: 1m
+      http_listen_port: {{ .Values.mimir.containerPorts.http }}
+      grpc_listen_port: {{ .Values.mimir.containerPorts.grpc }}
+    store_gateway:
+      sharding_ring:
+        wait_stability_min_duration: 1m
+        tokens_file_path: {{ .Values.mimir.dataDir }}/tokens
+    {{- if .Values.ruler.enabled }}
+    ruler:
+      alertmanager_url: dnssrvnoa+http://_http-metrics._tcp.{{ include "grafana-mimir.alertmanager.fullname" . }}-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}/alertmanager
+      enable_api: true
+      rule_path: {{ .Values.mimir.dataDir }}/ruler
+    ruler_storage:
+      {{- if .Values.minio.enabled }}
+      backend: s3
+      s3:
+        access_key_id: ${MIMIR_MINIO_ACCESS_KEY_ID}
+        secret_access_key: ${MIMIR_MINIO_SECRET_ACCESS_KEY}
+        bucket_name: ruler
+        endpoint: "{{ include "grafana-mimir.minio.fullname" . }}:{{ .Values.minio.service.ports.api }}"
+        insecure: {{ not .Values.minio.tls.enabled }}
+      {{- else }}
+      backend: {{ .Values.ruler.blockStorage.backend }}
+      {{ .Values.ruler.blockStorage.backend }}:
+        {{- include "common.tplvalues.render" (dict "value" .Values.ruler.blockStorage.config "context" $) | nindent 4 }}
+      {{- end }}
+    {{- end }}
+    {{- if .Values.alertmanager.enabled }}
+    alertmanager:
+      data_dir: {{ .Values.mimir.dataDir }}/alert-manager
+      enable_api: true
+      external_url: {{ .Values.mimir.httpPrefix.prometheus }}
+    {{- if .Values.minio.enabled }}
+    alertmanager_storage:
+      backend: s3
+      s3:
+        access_key_id: ${MIMIR_MINIO_ACCESS_KEY_ID}
+        secret_access_key: ${MIMIR_MINIO_SECRET_ACCESS_KEY}
+        bucket_name: ruler
+        endpoint: "{{ include "grafana-mimir.minio.fullname" . }}:{{ .Values.minio.service.ports.api }}"
+        insecure: {{ not .Values.minio.tls.enabled }}
+    {{- end }}
+    {{- end }}
+
+  ## @param mimir.overrideConfiguration [object] Mimir components configuration override. Values defined here takes precedence over mimir.configuration
+  ## e.g:
+  ## overrideConfiguration:
+  ##   auth_enabled: true
+  ##
+  overrideConfiguration: {}
+  ## @param mimir.existingConfigmap Name of a ConfigMap with the Mimir configuration
+  ##
+  existingConfigmap: ""
+  ## @param mimir.httpPrefix.prometheus HTTP URL path under which the Prometheus api will be served.
+  ## @param mimir.httpPrefix.alertmanager HTTP URL path under which the Alertmanager ui and api will be served.
+  ##
+  httpPrefix:
+    prometheus: /prometheus
+    alertmanager: /alertmanager
```

Comment:
This value does not seem to be used

Reply:
They are used here for example:
https://github.com/bitnami/charts/blob/3b08d9971ec28ca4ce362ac659b1360e432bd6e9/bitnami/grafana-mimir/templates/gateway/configmap-http.yaml#L111-L114